commands.adoc

```diff
@@ -223,10 +223,18 @@ Similarly, you can use `$ vagrant service-manager status openshift`,
 
 If you need to install the client binary for a specified service, use: +
 
-`vagrant service-manager install-cli [service]` +
+`vagrant service-manager install-cli [service] [options] [arg]` +
```

Comment:
@hferentschik `--cli-version=` will create more confusion IMO as user will then assume they need to enter option value as `--cli-version=1.3.0` which VSM cannot parse ATM.

Or you think the options description should include the value as:

The following options can be used: +

`--cli-version value`::  Specify the version of the binary to be installed +
`--path value`::   Specify the absolute or relative path where the binary should be installed +
`-h, --help`::   Prints help for the specific command being run.